doc/fitting.rst

```diff
@@ -15,59 +15,17 @@ when using complicated constraints or comparing results from related fits.
 
 
 The :func:`minimize` function
-===============================
+=============================
+
+.. currentmodule:: lmfit.minimizer
```

Comment:
> In general, I would favor cleaner/shorter internal links over shorter
intersphinx links

OK.  One of the problems with lmfit documentation in general is that the
external intersphinx links aren't consistent. E.g. one might have to use
`lmfit::Parameters` but `lmfit.Minimizer.minimize`. It's impossible to
predict which one. One has to inspect the objects.inv file to see which one
to use.